src/reset-password/ResetPasswordPage.jsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button, Input, ValidationFormGroup } from '@edx/paragon';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+
+import messages from './messages';
+import { resetPassword, validateToken } from './data/actions';
+import { resetPasswordResultSelector } from './data/selectors';
+import InvalidTokenMessage from './InvalidToken';
+import ResetSuccessMessage from './ResetSuccess';
+
+const Spinner = () => (
```

Comment:
Create it's own component so that we can use this in other components as well.

Reply:
Here is an example of how it's implemented in accounts MFE [PageLoading.jsx](https://github.com/edx/frontend-app-account/blob/8020ff58b8e8e8b73b066a012ab2da65213d394b/src/account-settings/PageLoading.jsx#L6)